### ISSUE TITLE
[sfml] Multiply blend mode + non-PMA doesn't look correct on Celestial Circus

### ISSUE THREAD

**Issue by jabuwu** [EsotericSoftware/spine-runtimes#2507]
Switching to `"data/celestial-circus.atlas"` here (removing `-pma`):
https://github.com/EsotericSoftware/spine-runtimes/blob/da05fb8f9237fd1a478ecd9f75cc4ca691146a3e/spine-sfml/c/example/main.cpp#L789

And setting this to `false`:
https://github.com/EsotericSoftware/spine-runtimes/blob/da05fb8f9237fd1a478ecd9f75cc4ca691146a3e/spine-sfml/c/example/main.cpp#L724

The mouth, nose, and eyes render with white outlines:
<img width="525" alt="Screen Shot 2024-04-25 at 4 29 53 AM" src="https://github.com/EsotericSoftware/spine-runtimes/assets/46233424/08cc2956-d469-4670-96a8-6f2a8e32d9e2">

Any insight why this might be happening? Blend modes are a source of great confusion for me. I generally use PMA and encourage others to do the same, so it's not a big deal, just curious if it might be a bug or some interaction I'm not familiar with.

Thanks for your time!

**Comment by badlogic** [comment]
The two slots in question use BlendMode:additive, which is defined like this in SFML

```
sf::BlendMode additive = sf::BlendMode(sf::BlendMode::SrcAlpha, sf::BlendMode::One);
```

This is correct in theory, however, for some reason, the rendering is not. I can only guess that this is a bug in SFML itself that is sadly outside our control.

TL;DR: keep using PMA :p
